test/Microsoft.AspNet.Razor.Runtime.Test/project.json

```diff
@@ -3,6 +3,7 @@
     "dependencies": {
         "Microsoft.AspNet.Razor.Runtime": "4.0.0-*",
         "Microsoft.AspNet.Testing": "1.0.0-*",
+        "Moq": "4.2.1312.1622",
```

Comment:
Bad practise to put a runtime-specific dependency in the general list. Move under frameworks / dnx451 / dependencies.